cmd/argo/commands/clustertemplate/create.go

```diff
@@ -1,60 +1,59 @@
-package clustertemplate
+package main
 
 import (
 	"context"
 	"log"
 	"os"
 
 	"github.com/argoproj/pkg/json"
-	"github.com/spf13/cobra"
-
-	"github.com/argoproj/argo-workflows/v3/cmd/argo/commands/client"
+	"github.com/argoproj/argo-workflows/v3/cmd/argo/commands/clustertemplate"
+	"github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1"
```

Comment:
`v1alpha1` is a very common name. if there's no reason to change it, it should be left as `wfv1` as it was before. `wfv1` is the convention in most of the codebase as well